tests/config/test_data.config

```diff
@@ -7,7 +7,6 @@ params {
             'genome' {
                 genome_fasta                                   = "${test_data_dir}/genomics/sarscov2/genome/genome.fasta"
                 genome_fasta_fai                               = "${test_data_dir}/genomics/sarscov2/genome/genome.fasta.fai"
-                genome_fasta_zip                               = "${test_data_dir}/genomics/sarscov2/genome/genome.fasta.zip"
```

Comment:
I generated it for the UNZIP module, but it isn't used for anything else. Given this PR includes a test data file that must be zipped anyway I thought it better to re-use that one rather than have two zip test dataset files.

Reply:
But then, you may need to change it also here, don't you?
https://github.com/jfy133/modules/blob/7fefef2fee90c91674bd15c525594546a5c674c2/tests/modules/unzip/main.nf#L9